src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -270,12 +290,20 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
       });
 
       try {
-        const facilityData = await getAdditionalFacilityInfo(
-          [].concat(...data),
-          featureFacilitiesServiceV2,
-        );
+        let facilityData;
+        if (
+          featureVAOSServiceVAAppointments ||
+          featureVAOSServiceCCAppointments
```

Comment:
I think you can leave off checking `featureVAOSServiceCCAppointments`, we don't use facility ids from those appointments.

Reply:
Done